src/test/java/posters/pageobjects/components/Header.java

```diff
@@ -22,20 +22,31 @@ public class Header extends AbstractComponent
     @Step("validate availability header")
     public void isComponentAvailable()
     {
-        $("#globalNavigation").should(exist);
+        $("#header-navigation-bar").should(exist);
     }
 
-    /// ----- validate header ----- ///
+    /// ----- validate header at homepage----- ///
```

Comment:
According to the code the sales banner is part of the main not of the header. So it would be nice to not have it in the header component. This would also resolve this "special method for special page" construct, which does not fit very nicely into the page concept.

Reply:
As a design recommendation,  we do not have the sales banner present throughout the website on other pages anymore like it used to be with the older posters. It only exists at the homepage.

Reply:
I understand. Removing sale banner from Header.java and Adding `SaleBanner.validateStructure();` to `Homepage.java`